app/javascript/components/App.js

```diff
@@ -24,13 +27,26 @@ class App extends React.Component{
             <NavLink to="/about">About Us</NavLink>
           </NavItem>
           <NavItem>
-            <NavLink to="/add">Add Apartment</NavLink>
+            <NavLink to="/new">Add Apartment</NavLink>
```

Comment:
I would consider moving these routes out of App.js and into whatever component you are using for navigation.